src/apps/uclient/startuclient.c

```diff
@@ -28,7 +28,9 @@
  * SUCH DAMAGE.
  */
 
+#if defined(__linux__)
```

Comment:
All the code before was using `__unix__`
Not sure what is the correct/best way (consistent ideally)

Reply:
See #1044 . 
After collecting everyone's needs, we will modify them later in a separate PR.